### PR TITLE
Import main node config file everytime node starts(simpler than removing config service usage)

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: #646

### CODE REVIEW

File: config/configstore/bootstrapper.go [hunk 1]
```diff
@@ -33,14 +33,12 @@ func (*Bootstrapper) Bootstrap(context map[string]interface{}) error {
 
 	nc := NewNodeConfig(cfg)
 	configdb.Register(nc)
-	_, err := service.GetConfig()
-	// if node config doesn't exist in the db, add it
+	// install the file based config everytime so that file updates are reflected in the db, direct updates to db are not allowed
+	nc, err := service.CreateConfig(NewNodeConfig(cfg))
```
Comment (vedhavyas) [inline config/configstore/bootstrapper.go]: So, this basically, overwrites the previous config ?

Comment (vimukthi-git) [reply config/configstore/bootstrapper.go]: yup

File: contextutil/context.go [hunk 2]
```diff
@@ -48,8 +48,8 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
 	return tc, nil
 }
 
-// CentContext updates a context with tenant info using the configstore, must only be used for api handlers
-func CentContext(ctx context.Context, cs config.Service) (context.Context, error) {
+// NodeContext updates a context with tenant info using the configstore, must only be used for api handlers
+func NodeContext(ctx context.Context, cs config.Service) (context.Context, error) {
```
Comment (mikiquantum) [inline contextutil/context.go]: I am not certain about this renaming, as Node sounds like a static context to me, and the purpose of this context is on request basis. I think it makes sense to just use `Context(...)`.

File: contextutil/context.go [hunk 3]
```diff
@@ -48,8 +48,8 @@ func Tenant(ctx context.Context) (config.TenantConfiguration, error) {
...
 	tcIDHex := ctx.Value(config.TenantKey).(string)
```
Comment (mikiquantum) [inline contextutil/context.go]: Can you create here the wrapping function you said you were going to do earlier?

Comment (vimukthi-git) [reply contextutil/context.go]: I think I didn't agree with that change since its not something that should be used commonly in the node.

**Pull request merged**